lib/src/Base/Type/openturns/SymmetricMatrix.hxx

```diff
@@ -154,6 +154,9 @@ public:
   Point computeEigenValues(const Bool keepIntact = true);
   Point computeEV(SquareMatrix & vOut,
                   const Bool keepIntact = true);
+  /** Compute an approximation of the largest eigenValue module using power iterations */
+  Scalar computeApproximateLargestEigenValueModule(const UnsignedInteger maximumIterations = ResourceMap::GetAsUnsignedInteger("Matrix-LargestEigenValueIterations"),
```

Comment:
missing `override`

Reply:
Could you recall me the meaning of the override keyword? I don't see why it should be put for this specific method and not for e.g. computeEV().

Reply:
`SymmetricMatrix` inherits from `SquareMatrix` and parent class has its own `computeApproximateLargestEigenValueModule` method. Thus you want to override the parent method and use this one for `SymmetricMatrix`

Reply:
Ok, I overlooked this point. I was shadowing the method instead of overloading it...